scanpy/logging.py

```diff
@@ -145,11 +130,12 @@ def print_versions():
     Matplotlib and Seaborn are excluded from this.
```

Comment:
The doctoring should probably be updated to something like: print versions of imported packages

Reply:
thank you, missed that one!